fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfiguration.java

```diff
@@ -28,7 +28,8 @@
     public static final String PROPERTY_DATASTORE_ID_HEADER_NAME = "fhirServer/core/datastoreIdHeaderName";
     public static final String PROPERTY_DEFAULT_TENANT_ID = "fhirServer/core/defaultTenantId";
     public static final String PROPERTY_DEFAULT_PRETTY_PRINT = "fhirServer/core/defaultPrettyPrint";
-    public static final String PROPERTY_HANDLING = "fhirServer/core/handling";
+    public static final String PROPERTY_DEFAULT_HANDLING = "fhirServer/core/defaultHandling";
+    public static final String PROPERTY_ALLOW_CLIENT_HANDLING_PREF = "fhirServer/core/allowClientHandlingPref";
```

Comment:
update to documentation in this PR? or in a subsequent PR?

Reply:
subsequent

Reply:
I'd vote to get it done in this PR